src/libs/actions/PaymentMethods.js

```diff
@@ -9,6 +10,35 @@ import * as Localize from '../Localize';
 import Navigation from '../Navigation/Navigation';
 import * as CardUtils from '../CardUtils';
 import ROUTES from '../../ROUTES';
+import NameValuePair from './NameValuePair';
+
+/**
+ * Deletes a debit card
+ *
+ * @param {Number} fundID
+ *
+ * @returns {Promise}
+ */
+function deleteDebitCard(fundID) {
+    return API.DeleteFund({fundID})
+        .then((response) => {
+            if (response.jsonCode === 200) {
+                Growl.show(Localize.translateLocal('paymentsPage.deleteDebitCardSuccess'), CONST.GROWL.SUCCESS, 3000);
+                Onyx.merge(ONYXKEYS.CARD_LIST, {[fundID]: null});
+            } else {
+                Growl.show(Localize.translateLocal('common.genericErrorMessage'), CONST.GROWL.ERROR, 3000);
+            }
+        })
+        .catch(() => {
+            Growl.show(Localize.translateLocal('common.genericErrorMessage'), CONST.GROWL.ERROR, 3000);
+        });
+}
+
+function deletePayPalMe() {
+    NameValuePair.set(CONST.NVP.PAYPAL_ME_ADDRESS, '');
```

Comment:
What happens if this fails? We don't catch it like other cases? Maybe I am missing that part when following the flow

Reply:
It looks like it would just fail. I will make a new issue to fix that as it looks like we don't return the call in `NVP.set` so none of our calls to that method have a failure case.